pkg/ostree/index.html

```diff
@@ -26,8 +26,13 @@
   <link href="ostree.css" rel="stylesheet">
   <script src="../base1/bundle.js"></script>
   <script src="ostree.js"></script>
+
+  <!-- debug:start -->
+  <script src="app.js"></script>
```

Comment:
Did the see the angular error though in one of the tests? I was figuring it was a loading race of some kind but maybe it was something else